Gemfile

```diff
@@ -1,19 +1,19 @@
 source 'https://rubygems.org'
 ruby "2.6.2"
 
-gem 'rails', '5.1.1'
+gem 'rails', '6.0.3.3'
 
 gem 'pg'
 
 gem 'coffee-script'
 gem 'uglifier'
-gem 'haml', '5.0.4'
-gem 'sass'
+gem 'haml', '~> 5.1.2'
+gem 'sassc'
```

Comment:
The `sass` gem is dead. `sassc` is the last hold out and needed for what's left of asset pipeline.